src/components/i18n/DopplerIntlProvider.test.tsx

```diff
@@ -1,33 +1,77 @@
+import { render, screen } from "@testing-library/react";
+import React from "react";
 import { DopplerIntlProvider } from "./DopplerIntlProvider";
-import { render } from "@testing-library/react";
-import "@testing-library/jest-dom/extend-expect";
 import { FormattedMessage } from "react-intl";
+import { useAppSessionState } from "../AppSessionStateContext";
+import { AppSessionState } from "../../abstractions/app-session";
 
-describe("DopplerIntlProvider", () => {
-  it("should render Campaign in English", () => {
-    const { getByText } = render(
-      <DopplerIntlProvider locale="en">
-        <FormattedMessage id="campaign" />
-      </DopplerIntlProvider>
-    );
-    getByText("Campaign");
-  });
+jest.mock("../AppSessionStateContext");
```

Comment:
I prefeer inject dependencies using `<AppServiceProvider>` in place of using jest's magic.

Reply:
You could inject a double of `AppSessionStateAccessor`.

Reply:
You create the double and inject it, let me know if you need help.

Reply:
✅